ansible/roles/common/tasks/main.yml

```diff
@@ -6,6 +6,16 @@
     # TODO create a flag for apt-get upgrade
     #upgrade: dist
 
+- name: Set up machine hostname
+  hostname:
+    name: "{{ liquid_domain }}"
```

Comment:
I think we should only set the hostname (basically `liquid_domain.split('.')[0]`). Fom `man hostname`:
```
FILES
       /etc/hostname  Historically  this file was supposed to only contain the
       hostname and not the full canonical FQDN.  Nowadays  most  software  is
       able  to  cope with a full FQDN here. This file is read at boot time by
       the system initialization scripts to set the hostname.

       /etc/hosts Usually, this is where one sets the domain name by  aliasing
       the host name to the FQDN.
```